concent_api/core/tests/test_auth_accept_or_reject_results.py

```diff
@@ -66,7 +66,7 @@ def test_provider_forces_subtask_results_for_task_which_was_already_submitted_co
             'core.message_handlers.payments_service.is_account_status_positive',
             side_effect=self.is_account_status_positive_true_mock
         ) as is_account_status_positive_true_mock_function:
-            with freeze_time("2018-02-05 10:00:30"):
+            with freeze_time("2018-02-05 10:00:29"):
```

Comment:
This change is not necessary if you change timestamp in below freeze_time

Reply:
right, fixed

Reply:
Looks like it's still not fixed.